starter-code/index.html

```diff
@@ -1,42 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
   <title>Spotify</title>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+<header>
+  <ul class="navbar">
+      <img class = "spotify-green-logo" src = "./images/spotify-logo.png" alt="spotify-logo">
+        <li href="#">Premium</li>
+        <li href="#">Discover</li> 
+        <li href="#">Help</li> 
+        <li href="#">Download</li>
+    </ul>
+</header>
+
+<div class = "banner">
+  <h1>Music for everyone.</h1>
+  <p>Spotify is now free on mobile, tablet and computer.<br>
+  Listen to the right music, wherever you are.</p>
+</div>
+
+<div class="whatson">
+
+  <h2>What’s on Spotify?</h2>
+
+  <div class = "column">
+
+    <div class = "music-icon">
+    <img src="images/music-icon.png" class="image-icon">
+    <h3>Millions of Songs</h3>
+    There are millions of<br>
+    songs on Spotify<br>
+    </div>
+
+    <div class = "hd-music">
+    <img src="images/high-quality-icon.png" class="high-quality-icon">
+    <h3>HD Music</h3>
+    Listen to music as if you<br>
+    were listening live<br>
+    </div>
+
+    <div class = "stream">
+    <img src="images/devices-icon.png" class="devices-icon">
+    <h3>Stream Everywhere</h3>
+    Stream music on your<br>
+    Smartphone, tablet or<br>
+    computer<br>
+    </div>
```

Comment:
Keep in mind that classes are meant to be reusable and ids should be unique.  Here you have three boxes that should be the same except for their content.  In this case I would try something like this : `<div class = "box">`
so that in your CSS you can give all your boxes a width of 33.33333%  without having to repeat yourself.  Then if you need to add specific attributes you can target the children of `.column` using the [nthChild selector]( https://www.w3schools.com/cssref/sel_nth-child.asp).  Or you can make the `.column` container a flex container and let it fill its space equally. 

Basically if you can think of better "general rules" to apply to your layout you will have an easier time and have to use few selectors and classes.